examples/example_mod.m

```diff
@@ -0,0 +1,23 @@
+% Initialization.
+clear;
+setup;
+n = neuron.Neuron();
+
+% Try to find 'hd' mechanism.
+axon1 = n.Section("axon1");
+axon1.insert_mechanism("hd");  % Warning: Insertable mechanism 'hd' not found. 
+
+% Compile mod file.
+examples_path = fileparts(mfilename('fullpath'));
+mod_path = fullfile(examples_path, 'mod');
+system("nrnivmodl " + mod_path);
+output_path = fullfile(pwd, 'nrnmech.dll');
+dll_path = fullfile(mod_path, 'nrnmech.dll');
+system("move " + output_path + " " + dll_path);
```

Comment:
This is windows-specific. Can we use MATLAB's [movefile](https://www.mathworks.com/help/matlab/ref/movefile.html) instead so that the example should work on all platforms?